client/src/index.tsx

```diff
@@ -1,18 +1,30 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
 import { HashRouter as Router } from 'react-router-dom';
+import * as Sentry from '@sentry/react';
+import { Integrations } from '@sentry/tracing';
 import 'typeface-berkshire-swash';
 import { MainRouter } from './router';
 import { setupFrontendListener } from 'eiphop';
 import { AppDataProvider } from './routes/AppDataProvider';
 import { StoreProvider } from 'easy-peasy';
 import { mainStore } from './store/mainStore';
+require('dotenv').config();
 
 (window as any).CSPSettings = {
     nonce: 'nonce',
 };
 const { ipcRenderer } = window as any;
 
+Sentry.init({ 
+    dsn: process.env.REACT_APP_SENTRY_DSN,
+    integrations: [new Integrations.BrowserTracing()],
+    tracesSampleRate: 1.0,
+    environment: process.env.NODE_ENV,
+});
+console.log('init');
```

Comment:
Move before line 19.
```suggestion
console.log('init Sentry');
```